dev-docs/roadmaps/lighting-roadmap.md

```diff
@@ -0,0 +1,27 @@
+# Lighting Roadmap
+
+
+
+
+## Motivation
+A flexible and featureful lighting module is highly expected by users, could bring rendering quality to a new level. 
+
+## Major Features
+
+
+### Different Light Sources Per Deck
+
+* Ambient light
```

Comment:
Maybe add a comment that we don't support per-layer lights. 

Idea: It could be possible to support per-view lights, instead of global lights, then we could add the light parameters to the `View` class.